frontend/src/vault/balance/RefreshVaultBalance.tsx

```diff
@@ -20,7 +22,12 @@ export const RefreshVaultBalance = () => {
         getCoinPricesQueryKeys(coins.map(getStorageCoinKey)),
         ...coins.map(coin =>
           getBalanceQueryKey({
-            ...getStorageCoinKey(coin),
+            ...getCoinMetaKey({
```

Comment:
The idea with getStorageCoinKey it that it will also have an address, so that should result in the same object is it was before:

```tsx
...getCoinMetaKey({
Comment
The idea with getStorageCoinKey it that it will also have an address, 
 
              ticker: coin.ticker,
              contractAddress: coin.contract_address,
              isNativeToken: coin.is_native_token,
              chain: coin.chain as Chain,
            }),
            address: coin.address,
```